docs/features/transactions/signatures.md

```diff
@@ -131,73 +131,24 @@ Extend the example from the [Multisignature Account](../accounts/create.md#multi
 ```javascript tab="JavaScript"
 const algosdk = require('algosdk');
 
-/**
- * utility function to wait on a transaction to be confirmed
- * the timeout parameter indicates how many rounds do you wish to check pending transactions for
- */
-const waitForConfirmation = async function (algodclient, txId, timeout) {
-    // Wait until the transaction is confirmed or rejected, or until 'timeout'
-    // number of rounds have passed.
-    //     Args:
-    // txId(str): the transaction to wait for
-    // timeout(int): maximum number of rounds to wait
-    // Returns:
-    // pending transaction information, or throws an error if the transaction
-    // is not confirmed or rejected in the next timeout rounds
-    if (algodclient == null || txId == null || timeout < 0) {
-        throw "Bad arguments.";
-    }
-    let status = (await algodclient.status().do());
-    if (status == undefined) throw new Error("Unable to get node status");
-    let startround = status["last-round"] + 1;
-    let currentround = startround;
-
-    while (currentround < (startround + timeout)) {
-        let pendingInfo = await algodclient.pendingTransactionInformation(txId).do();
-        if (pendingInfo != undefined) {
-            if (pendingInfo["confirmed-round"] !== null && pendingInfo["confirmed-round"] > 0) {
-                //Got the completed Transaction
-                return pendingInfo;
-            }
-            else {
-                if (pendingInfo["pool-error"] != null && pendingInfo["pool-error"].length > 0) {
-                    // If there was a pool error, then the transaction has been rejected!
-                    throw new Error("Transaction Rejected" + " pool error" + pendingInfo["pool-error"]);
-                }
-            }
-        }
-        await algodclient.statusAfterBlock(currentround).do();
-        currentround++;
-    }
-    throw new Error("Transaction not confirmed after " + timeout + " rounds!");
+const waitForConfirmation = async function (algodClient, txId, timeout) {
+    // ...covered in a previous example
```

Comment:
I feel this method must be included within this script as the reader may not access related content and we want the code to be complete inline.